modules/articles/client/controllers/admin/article.client.controller.js

```diff
@@ -21,6 +21,7 @@
     function remove() {
       if ($window.confirm('Are you sure you want to delete?')) {
         vm.article.$remove($state.go('admin.articles.list'));
+        Notification.success('Article deleted successfully!');
```

Comment:
@sujeethk After merging #1534 this is creating a merge conflict. It's probably best to put this in the `$remove` callback. Can you rebase, and this line?